.gitignore

```diff
@@ -19,3 +19,7 @@ yarn.lock
 
 # TypeScript
 jsconfig.json
+
+# JetBrains
```

Comment:
I don't ignore it in every project; it's quite helpful to me for projects where I actually want the JetBrains IDE files (usually not ones I'm sharing), which is why I don't have it in global. I *can* ignore it at that level, but as soon as someone *else* adds it and doesn't have that… ¯\\\_(ツ)_/¯ I'm a fan of "ignore everything like this instead of accidentally committing it and being sad"; but that's me.